<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSwashplateWidget</class>
 <widget class="QWidget" name="QSwashplateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>十字盘类型</string>
   </property>
   <widget class="QRadioButton" name="radioButton_H1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>H1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_H3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>H3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_HR3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HR-3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_H140">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>70</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>H140</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_HE3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HE3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_H4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>H4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TILTTYPESET">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/set.png</normaloff>:/images/set.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TILTTYPEASK">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>64</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/ask.png</normaloff>:/images/ask.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_37">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="title">
    <string>旋转</string>
   </property>
   <widget class="QPushButton" name="pushButton_TILTPHASESET">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/set.png</normaloff>:/images/set.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_phase">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>36</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> 0 °</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_phase">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TILTPHASEASK">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/ask.png</normaloff>:/images/ask.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_72">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>471</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>十字盘混控比</string>
   </property>
   <widget class="QPushButton" name="pushButton_TILTDIRSET">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/set.png</normaloff>:/images/set.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TILTDIRASK">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/ask.png</normaloff>:/images/ask.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>22</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>俯仰</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ELEDIRECT">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>22</y>
      <width>64</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>正向</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/dir1.png</normaloff>:/images/dir1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AILDIRECT">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>50</y>
      <width>64</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>正向</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/dir1.png</normaloff>:/images/dir1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>50</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>滚转</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>80</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>总距</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_COLDIRECT">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>80</y>
      <width>64</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>正向</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/dir1.png</normaloff>:/images/dir1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_MIXELE">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>16</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string> 60% </string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_MIXELE">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>60</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_MIXAIL">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>60</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_MIXCOL">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>76</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string> 60% </string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_MIXCOL">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>60</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_MIXAIL">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>46</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string> 60% </string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_115">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>260</y>
     <width>108</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请完成十字盘的配置</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_s">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>471</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>十字盘舵机</string>
   </property>
   <widget class="QLabel" name="label_116">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_117">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_118">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_119">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_120">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>方向</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_S1DIR">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>45</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>正向</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/dir1.png</normaloff>:/images/dir1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_S2DIR">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>76</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>正向</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/dir1.png</normaloff>:/images/dir1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_S3DIR">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>107</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>正向</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/dir1.png</normaloff>:/images/dir1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_S4DIR">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>136</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>正向</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/dir1.png</normaloff>:/images/dir1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_S1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>46</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_S2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>79</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_S3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>109</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_S4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>139</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_121">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>微调</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_S1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>42</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string> 0 </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_S2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>74</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string> 0 </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_S3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>104</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string> 0 </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_S4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>134</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string> 0 </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TILTS1ASK">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/ask.png</normaloff>:/images/ask.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TILTS1SET">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/set.png</normaloff>:/images/set.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TILTS2SET">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>74</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/set.png</normaloff>:/images/set.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TILTS2ASK">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>74</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/ask.png</normaloff>:/images/ask.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TILTS3SET">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>108</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/set.png</normaloff>:/images/set.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TILTS3ASK">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>108</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/ask.png</normaloff>:/images/ask.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TILTS4SET">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/set1.png</normaloff>:/images/set1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TILTS4ASK">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/ask.png</normaloff>:/images/ask.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_TILTTYPE">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>321</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../qfi.qrc">:/qfi/images/H1.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../qfi.qrc"/>
 </resources>
 <connections/>
</ui>
